IndyAV/simulation/indyav_gazebo/xacro/wheels.xacro

```diff
@@ -2,93 +2,99 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro" name="wheels">
   <xacro:include filename="$(find mil_gazebo)/xacro/util.xacro"/>
 
-  <xacro:property name="wheel_length" value="0.25"/>
-  <xacro:property name="wheel_width" value="0.25"/>
-  <xacro:property name="wheel_height" value="0.25"/>
-  <xacro:property name="wheel_mass" value="5"/>
+  <xacro:property name="front_wheel_radius" value="0.127"/>
+  <xacro:property name="front_wheel_width" value="0.127"/>
+  <xacro:property name="front_wheel_mass" value="3"/>
+  
+  <xacro:property name="back_wheel_radius" value="0.1335"/>
+  <xacro:property name="back_wheel_width" value="0.203"/>
+  <xacro:property name="back_wheel_mass" value="5"/>
 
-  <xacro:macro name="wheel_geometry">
+  <xacro:macro name="front_wheel_geometry">	  
     <geometry>
-      <box size="${wheel_length} ${wheel_width} ${wheel_height}"/>
+       <cylinder length="${front_wheel_width}" radius="${front_wheel_radius}"/>
     </geometry>
   </xacro:macro>
 
-  <xacro:macro name="wheel_surface">
-    <surface>
-      <friction>
-        <ode>
-          <mu>1000</mu> <!-- left / right  friction ,very high -->
-          <mu2>1</mu2> <!-- forwards / backwards friction, very low -->
-          <fdir1> 0 1 0 </fdir1>
-        </ode>
-      </friction>
-    </surface>
+  <xacro:macro name="back_wheel_geometry">
+    <geometry>
+       <cylinder length="${back_wheel_width}" radius="${back_wheel_radius}"/>
+    </geometry>
   </xacro:macro>
 
   <xacro:macro name="do_wheel_surface" params="name">
```

Comment:
this macro is defined both here and on line line 63, it may cause issues in the future.